kotlinx.fuzz.gradle/src/main/kotlin/kotlinx/fuzz/gradle/KFuzzPlugin.kt

```diff
@@ -18,6 +18,8 @@ import org.gradle.api.tasks.options.Option
 import org.gradle.api.tasks.testing.Test
 import org.gradle.kotlin.dsl.*
 
+private const val INTELLIJ_DEBUGGER_DISPATCH_PORT_PROPERTY = "idea.debugger.dispatch.port"
```

Comment:
I don't like the fact that we have more and more duplicated code and constants. Maybe we should give up and make one big `Constants` class somewhere in the api or engine. I think @DLochmelis33 has it already in #37, maybe we should make a "great refactoring" after it is merged

Reply:
Sounds reasonable, let's merge this PR after that one, so that I will have my code formatted